lms/static/sass/discussion/views/_inline.scss

```diff
@@ -35,7 +35,6 @@
 
     .wmd-preview-container {
         @include discussion-new-post-wmd-preview-container;
-        margin-bottom: $baseline;
```

Comment:
relating to the preview area, it would be great to either eliminate the gap between this and the text input area (since that is the reason for the heavy border and selective border radii as shown below)

Current
![image](https://cloud.githubusercontent.com/assets/2023680/21165123/5ddec6a6-c16b-11e6-98dd-1f6b1f1cfaf3.png)

Or we could update the border radii and border to what is shown below if that is easier.
![image](https://cloud.githubusercontent.com/assets/2023680/21165136/74dcf3fa-c16b-11e6-911f-d4ca9c1dd2b1.png)